src/seedu/addressbook/parser/Parser.java

```diff
@@ -96,7 +96,7 @@ public Command parseCommand(String userInput) {
      * @param args full command args string
      * @return the prepared command
      */
-    private Command prepareAdd(String args){
+    private static Command prepareAdd(String args){
```

Comment:
The purpose here is to understand when you need to have a class level method. So do you think all the methods in Parser class need to be class-level? Think about it.